src/patternfly/components/Form/examples/form-text-input-example.hbs

```diff
@@ -1,39 +1,39 @@
 {{#> form}}
   {{#> form-group}}
     {{#> form-label for="textInput1"}}Standard input{{/form-label}}
-    {{#> form-controls controlType="input" input="true" type="text" name="inputName1" id="textInput1"}}{{/form-controls}}
+    {{#> form-controls controlType="input" input="true" type="text" name="textInput1" id="textInput1"}}{{/form-controls}}
   {{/form-group}}
   {{#> form-group}}
     {{#> form-label for="textInput2"}}Input with placeholder{{/form-label}}
     {{#> form-controls controlType="input" input="true" type="text" name="textInput2" id="textInput2" placeholder="Placeholder input example"}}{{/form-controls}}
   {{/form-group}}
     {{#> form-group}}
-    {{#> form-label for="textInput2"}}Readonly Input{{/form-label}}
-    {{#> form-controls controlType="input" input="true" type="text" name="textInput2" id="textInput2" readonly="true" value="Readonly input example"}}{{/form-controls}}
+    {{#> form-label for="textInput3"}}Readonly Input{{/form-label}}
+    {{#> form-controls controlType="input" input="true" type="text" name="textInput3" id="textInput3" readonly="true" value="Readonly input example"}}{{/form-controls}}
   {{/form-group}}
```

Comment:
The readonly input currently allows text input. Need to add `readonly` to input

Reply:
doh